src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -31,34 +31,35 @@
             <label class="form-label col">Reports</label>
             <table class="table-bordered col-10">
                 <tr v-for="report in workflowRunStatus.data.reports">
-                    <td v-if="report.status === 'success'">
+                    <td v-if="report.status === 'success'" class="p-2">
                         <a
                             :href="
                                 reportVersionHref(report.name, report.version)
                             "
                             >{{ report.name }}</a
                         >
                     </td>
-                    <td v-else-if="report.status === 'error'">
+                    <td v-else-if="report.status === 'error'" class="p-2">
                         <a
                             :href="reportLogsHref(report.name)"
                             @click="setReportLogsTab"
                             >{{ report.name }}</a
                         >
                     </td>
-                    <td v-else>{{ report.name }}</td>
-                    <td :class="statusColour(report.status)">
+                    <td v-else class="p-2">{{ report.name }}</td>
+                    <td :class="statusColour(report.status)" class="p-2">
                         {{ interpretStatus(report.status) }}
                     </td>
-                    <td v-if="report.date">{{ formatDate(report.date) }}</td>
+                    <td v-if="report.date" class="p-2">{{ formatDate(report.date) }}</td>
                 </tr>
             </table>
         </div>
-        <div class="row justify-content-end mt-3">
-            <button class="button mr-3" type="button" disabled>
+        <!-- Buttons to be unhidden and made active once workflow wizard is implemented -->
```

Comment:
Is there a ticket for this? The wizard is pretty much done now, so that should be possible to do.

Reply:
mrc-2513